src/components/hero-image/hero-image.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import heroImage from '~/assets/img/heroImage.jpg';
+import styles from './hero-image.module.scss';
+import { StretchableLink } from '../stretchable-link/stretchable-link';
+
+export interface HeroImageProps {
+    subTitle: string;
+    title: string;
+    className?: string;
+    linkLabel?: string;
+    linkCategorySlug?: string;
+}
+
+export const HeroImage = ({
```

Comment:
Don't create this component, just put the JSX in the home screen. You're forcing them to link to a category and they can't change it without editing the code of this component manually.